File: QuizUI/src/app/components/home/entities/create-option.model.ts
export class CreateOptionModel {
  questionId: number;
  optionText: string;
  isCorrect: boolean;

  constructor(questionId: number, optionText: string, isCorrect: boolean) {
    this.optionText = optionText;
    this.questionId = questionId;
    this.isCorrect = isCorrect;
  }
}

File: BLL/Models/OptionModels/CreateOptionModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace BLL.Models.OptionModels
{
    public class CreateOptionModel
    {
        public int QuestionId { get; set; }
        public string OptionText { get; set; }
        public bool IsCorrect { get; set; }
    }
}
